### ISSUE TITLE
Re-enabling dropout in fine-tuning when using t5.1.1.xl model

### ISSUE THREAD

**Issue by Halli20** [google-research/text-to-text-transfer-transformer#259]
I'm using the t5.1.1.xl model and i first train it further on several supervised tasks and then fine-tune it on a specific task. In the released checkpoint folder you indicate that you would suggest to re-enable dropout when fine-tuning the model. How do I re-enable dropout because in the configuration file it seems that it is already enabled?

**Comment by craffel** [comment]
If you use the operative config for t5.1.1.whatever, dropout will be enabled for you already via the operative config. You don't need to do anything else.
